targets/ath79-generic

```diff
@@ -356,6 +356,7 @@ device('tp-link-tl-wr1043nd-v4', 'tplink_tl-wr1043nd-v4', {
 	},
 })
 
+device('tp-link-wbs210-v1', 'tplink_wbs210-v1')
```

Comment:
You mean `'tp-link-wbs210-v1.20',`?

I thought that was an even older manifest alias back then.
https://github.com/freifunk-gluon/gluon/commit/45c84a117bf820bf13e72187b46321b959e23aed#diff-ce406397f31c274de9f8a5907eaf20b0194cccbb6995fa59490e0c56dbd3dea5L292-L294

Will add it in a moment.

Reply:
All the TP-Link CPE and WBS devices have model names with precise version numbers in ar71xx.

Reply:
resolved in 9af3047994a8a3833adaa9d6ddf9a177370e1fe3

Reply:
Weird that that was not an issue in @rotanid's testing of sysupgrade then?

Reply:
The name is only used by the autoupdater to find out which manifest entry to use. It has absolutely no effect on manual sysupgrades.

One way to check for this would be to run the image name commend both before and after the upgrade.